docs/guide/configuration/devices-groups.md

```diff
@@ -43,6 +43,17 @@ Every Zigbee Device supports the following list of options.
 **`friendly_name`**  
 Used in the MQTT topic of a device. By default, this is the device ID (e.g. `0x00128d0001d9e1d2`).
 
+::: tip
+You can use the `/` separator in `friendly_name` to _group_ devices.  
+For example, using a friendly-name like `Kitchen/Floor Light` would result in a corresponding MQTT structure with `Kitchen` as folder containing `Floor Light`.
+:::
+
+::: warning
+Note that a `friendly_name` is **NOT** allowed to end with `/` or use one of the
```

Comment:
It is not allowed to end with a `/` or `/` with one of the endpoint names (e.g. `/left` is not allowed)